interfaces/HO-Portal/src/global.scss

```diff
@@ -97,6 +97,15 @@ ion-button {
       --padding-end: 8px;
     }
   }
+
+  .validation-message {
+    min-height: 36px;
```

Comment:
To avoid using a 'magical' number/height here, we could make the text 'invisible' instead.
So the text will always be rendered AND occupy space/height, regardless of its size/length(taking translations into account).

So the default-styles will be: 
```css
.validation-message {
  // By default 'invisible':
  visibility: hidden;

  &.is-triggered {
    visibility: visible;
  }
}
```